desktop/linux/index.md

```diff
@@ -24,10 +24,23 @@ To install Docker Desktop for Linux:
 1. Set up the [Docker repository](../../engine/install/ubuntu.md#install-using-the-repository).
 2. Download and install the Tech Preview Debian package:
     ```console
-    $ curl https://desktop-stage.docker.com/linux/main/amd64/73772/docker-desktop.deb --output docker-desktop.deb
+    $ curl https://desktop-stage.docker.com/linux/main/amd64/74134/docker-desktop.deb --output docker-desktop.deb
     $ sudo apt install ./docker-desktop.deb
     ```
-3. Check whether the user belongs to `docker` and `kvm` groups. You may need to restart the host to load the group configuration.
+3. Check whether the user belongs to `docker` and `kvm` groups. You may need to restart the host to load the group configuration (automated in post-install script). 
```

Comment:
Is there no way for the user to just reload his groups ?
Traditionnally `su $(whoami)` should open a shell with the new groups available.

Reply:
We are already doing a daemon reload in the post-install script. We wanted just to have users check that worked.

Reply:
You know like me that we don't like to reboot our linux machines 😄 
There are two parts : the system service, and the user access to the services.
So for the systemd service, I don't know if systemd is able to handle the new groups as the user is added into them or if the user need to be member of the group before the session service starts. But in that case, a logout/login cycle should be sufficient.
And for the user access to the services `su $USER` should do the trick.
But I agree I haven't tested it